lib/Unicode/UCD/Core.hs

```diff
@@ -16,26 +16,38 @@ where
 import qualified Unicode.Internal.Generated.DerivedCoreProperties as P
 import qualified Unicode.Internal.Generated.PropList as P
 
--- | Selects Alphabetic Unicode characters defined in Unicode character
--- database.  This should be equivalent to "Data.Char.isAlpha".
+-- | Returns 'True' for alphabetic Unicode characters (lower-case, upper-case
+-- and title-case letters, plus letters of caseless scripts and modifiers
+-- letters).
+--
+-- prop> isAlpha == Data.Char.isAlpha
```

Comment:
Wrap this in code blocks? `@`

Reply:
It is recognized by haddock as it is and rendered in a box.